app/assets/javascripts/my_modules/archived.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  let selectedTasks = [];
+
+  $('#module-archive').on('click', '.restore-button-container', function(e) {
+    e.preventDefault();
+    alert('woop! Not implemented yet!')
+  });
+
+  $('#module-archive').on('click', '.task-selector', function() {
+    let taskId = $(this).closest('.task-selector-container').data('task-id');
+    let index = $.inArray(taskId, selectedTasks);
+    let resoteTasksButton = $('.restore-button-container');
+
+    // If checkbox is checked and row ID is not in list of selected folder IDs
+    if (this.checked && index === -1) {
+      selectedTasks.push(taskId);
+      resoteTasksButton.collapse('show');
```

Comment:
just - `.show() / .hide()`

Reply:
It's not smooth BS collapse animations. It jumps as a toggling display property